addons/dialogue_nodes/editor/graph.gd

```diff
@@ -166,6 +170,9 @@ func disconnect_node_signals(node : GraphNode):
 			characters_updated.disconnect(node._on_characters_updated)
 			node.disconnection_from_request.disconnect(_on_disconnection_from_request)
 			node.connection_shift_request.disconnect(_on_connection_shift_request)
+		6: # fork node
```

Comment:
Is this required?

Reply:
Yes, explanation on the review above this one.

Reply:
Yup, we're keeping this code.